src/schema/v1.json

```diff
@@ -64,6 +64,10 @@
              "image",
              "url"
          ],
+         "optional": [
```

Comment:
Since the "project" and the "family" are unique GCP, this should be a property of the GCP image itself otherwise it would become valid for all Cloud Artifacts.